Dockerfile

```diff
@@ -1,145 +1,166 @@
 #--------------------------------------
 # Target image to build
 #--------------------------------------
-ARG IMAGE=latest
+ARG TARGET=latest
 
 #--------------------------------------
 # Ubuntu base image to use
 #--------------------------------------
-ARG FLAVOR=latest
+ARG FLAVOR=bionic
+ARG BASE_IMAGE=${FLAVOR}
```

Comment:
Shouldn't this be `ubuntu:${FLAVOR}`?

Reply:
stage names can't contain `:` so that's why i removed it.

Reply:
OK, I hadn't realized it was tied to the rebuild trigger aliases